src/components/loadable-item.cjsx

```diff
@@ -18,9 +18,25 @@ module.exports = React.createClass
     renderItem: React.PropTypes.func.isRequired
     saved: React.PropTypes.func
     load: React.PropTypes.func
+    renderLoading: React.PropTypes.func
+    update: React.PropTypes.func
+
+  componentDidMount: -> @reload({})
+  componentDidUpdate: (oldProps) -> @reload(oldProps)
```

Comment:
That is the time at which the destination loadable needs to reload -- i.e. when the loadable for homework gets mounted and you are working from problem to problem, props will update, but the store for the step may not necessarily need to.  when we go from making progress on the homework back to the dashboard or anywhere else, the destination's loadable store needs to check the server for updates